import-export-cli/integration/testutils/environment_testUtils.go

```diff
@@ -114,8 +114,15 @@ func ValidateExportApiPassed(t *testing.T, args *ApiImportExportTestArgs, direct
 	//Derive exported path from output
 	exportedPath := filepath.Dir(base.GetExportedPathFromOutput(output))
 
+	//Check directory path to omit changes due to OS differences
+	if strings.Contains(directoryName, ":") {
+		slittedString := strings.SplitAfter(directoryName, ":")
+		directoryName = slittedString[1]
+	}
+
 	assert.True(t, strings.HasPrefix(exportedPath, directoryName), "API doesn't seem to exported to the "+
-		"changed exported location. Expected in "+directoryName+" but it is exported in "+exportedPath)
```

Comment:
Lets change this to something like,

`assert.True(t, strings.HasPrefix(exportedPath, directoryName), "API export path "+exportedPath+" is not within the expected export location "+directoryName)`

This makes it very clear what the expectation is

Reply:
Fixed

Reply:
Fixes